### PR TITLE
let's start

### PR DESCRIPTION
**Pull request by AnnaRokh**

### CODE REVIEW

File: src/package1/Class1.java
Comment (tohakim29) [inline src/package1/Class1.java:1]: jamshed added changes to Class1.java

Comment (PRiv232) [reply src/package1/Class1.java:1]: WE work together nothing to hide here

Comment (AnnaRokh) [reply src/package1/Class1.java:1]: okay, let someone else write something

**Comment by suleinanir** [comment]
Great ANNA!

File: src/package1/Class1.java
Comment (jamaqodir) [reply src/package1/Class1.java:1]: who is available to make clone?

Comment (AnnaRokh) [reply src/package1/Class1.java:1]: should work

**Comment by jamaqodir** [comment]
Sule merged Pull request successfully

**Pull request merged**